test/stream/api/sink/rollback-version-ok/005-patch-settings/request.json

```diff
@@ -0,0 +1,20 @@
+{
+  "path": "/v1/branches/default/sources/my-source/settings",
```

Comment:
You're modifying source settings instead of sink settings here... I noticed this because in `009-list-sink-versions` you only have two versions instead of three.